### DISCUSSION TITLE
Please share your MM sites beyond project documentation!

### DISCUSSION THREAD

**Discussion by rktic** [squidfunk/mkdocs-material#6151] [Show and tell]
Hej,

I'm keen to learn how others use mkdocs Material for their sites. Especially in the realm of personal knowledge management, second brains/digital gardens. Struggling to decide on structure for the vast array of inherently different topics I'm dealing with. And would very much appreciate the inspiration :)

Thank you in advance!
Ronny

**Comment by McDic** [comment]
You can see my blog made with `mkdocs-material`. https://github.com/squidfunk/mkdocs-material/discussions/6631

**Comment by shenweiyan** [comment]
This is my website <https://www.weiyan.cc/> where I've integrated both personal knowledge management and a blog with `mkdocs-material`.

**Comment by MaximilianKohler** [comment]
I used MM for a personal website, even though it's not ideal, just because I didn't want to spend time figuring out something new, and I haven't come across an ideal SSG for personal/business sites.

It's https://michaelharrop.com/.

I just came across this tutorial for setting up a homepage, but for me it's not worth the time & effort. https://github.com/squidfunk/mkdocs-material/discussions/7838

**Comment by santaimpersonator** [comment]
You can find a list of our docs here: https://docs.sparkfun.com/

- We are working on updating the look of that landing page
- Docs list should be sorted by latest commits

**Comment by SilentGlasses** [comment]
My Getting Started with Linux site _(there may be some grammatical or formatting issues but it's a work in progress)_:
https://silentglasses.github.io/linux_start/
